packages/core/src/routes/organization-invitation/index.ts

```diff
@@ -94,11 +96,20 @@ export default function organizationInvitationRoutes<T extends ManagementApiRout
       body: sendMessagePayloadGuard,
       status: [204],
     }),
+    koaEmailI18n(queries),
     async (ctx, next) => {
       const {
         params: { id },
         body,
       } = ctx.guard;
+      assertThat(
```

Comment:
Add some comments like, make typescript happy, since the koaRoute can not infer the context type properly with koaEmathe ilI18n middleware applied ahead.